tests/flux-v2-image-automation.sh

```diff
@@ -49,14 +49,32 @@ for FILE_LOCATION in $(echo ${FILE_LOCATIONS}); do
             fi
 
             IMAGE_AUTOMATION_CHECK=$(cat imagepolicies_list.yaml  | \
-            IMAGE_POLICY_NAME="${IMAGE_POLICY}" yq eval 'select(.metadata and .kind == "ImagePolicy" and .metadata.name == env(IMAGE_POLICY_NAME) )' - | yq eval '.spec.filterTags.pattern == "^prod-[a-f0-9]+-(?P<ts>[0-9]+)"' -)
+            IMAGE_POLICY_NAME="${IMAGE_POLICY}" yq eval 'select(.metadata and .kind == "ImagePolicy" and .metadata.name == env(IMAGE_POLICY_NAME) )' - | yq eval '.spec.filterTags.pattern | "^prod-[a-f0-9]+-(?P<ts>[0-9]+)"' -)
 
-            if [ $IMAGE_AUTOMATION_CHECK == false ]
-            then
-                echo "Non whitelisted pattern found in ImagePolicy: $IMAGE_POLICY it should be ^prod-[a-f0-9]+-(?P<ts>[0-9]+)" && exit 1
+                if [[ $IMAGE_AUTOMATION_CHECK == "true" ]]; then
+                    echo "Non whitelisted pattern found in ImagePolicy: $IMAGE_POLICY it should be ^prod-[a-f0-9]+-(?P<ts>[0-9]+)"
+                    exit 1
+                fi
+            done
+
+            # IMAGE_REPOSITORY_REF=$(yq e '.spec.imageRepositoryRef.name' imagepolicies_list.yaml)
+            PATTERN="^prod-[a-f0-9]+-(?P<ts>[0-9]+)"
+            # FILE="clusters/prod/base"
+            FILE="./apps/juror/juror-api/juror-api.yaml"
+
+            # for file in $FILE/*.yaml; do
+
+            IMAGE_TAG=$(yq eval '.spec.values.java.image' "$FILE")
```

Comment:
The image tag here will be the image name + the tag so for example: `sdshmctspublic.azurecr.io/juror/api:prod-a6bc219-20240226123038` so you'll need to add something that pulls out only the info after the `:` as thats the tag